phpBB/phpbb/cache/driver/null.php

```diff
@@ -15,137 +15,8 @@
 
 /**
 * ACM Null Caching
+* @deprecated 3.1.5 (to be removed: 3.3.0)
 */
-class null extends \phpbb\cache\driver\base
+class null extends \phpbb\cache\driver\dummy
```

Comment:
I think we can drop this file, since the value is coded into the config file we need to add a "workaround" there anyway, like with the namespaces: https://github.com/phpbb/phpbb/blob/3.1.x/phpBB/phpbb/di/extension/config.php#L80-L88